lib/phoenix/channel/transport.ex

```diff
@@ -119,7 +119,7 @@ defmodule Phoenix.Channel.Transport do
 
     * `topic` - The String value "phoenix"
     * `event` - The String value "heartbeat"
-    * `payload` - An empty JSON message payload, ie {}
+    * `payload` - An empty JSON message payload, that is {}
```

Comment:
This one is fine. We just need to put ticks around `{}`. Another approach is:

     An empty JSON message payload (`{}`)